types/got/index.d.ts

```diff
@@ -289,6 +289,8 @@ declare namespace got {
         fromCache: boolean;
         redirectUrls?: string[];
         retryCount: number;
+        statusCode: number;
```

Comment:
nit: Maybe add some comment for this? For example

> got's Response is always a "response obtained from http.ClientRequest", therefore these two properties are not `undefined`

Reply:
Fixed 👌